### ISSUE TITLE
(javascript)  Once the name of a function starts with 'get' or 'set', it will generate an erroneously highlighted result

### ISSUE THREAD

**Issue by chuyueZhang** [highlightjs/highlight.js#2548]
This is the code I typed and I find that the function name and the comment are highlighted incorrectly when I named function with 'get(anything)' or 'set(anything)'.
```
  async function test(){
      let result = await getNews('bbb')   //await后的函数只有promise为成功状态才会继续执行
      //此处的if判断是建立在上个函数调用resolve的基础上
      if(result===false){ //可以借用给resolve传递false来表示promise实际为fullfilled状态
          console.log('async函数失败') //针对fullfilled状态需要执行的逻辑
      }else{
          console.log(result) //成功状态执行的逻辑
      }
      result = await getNews('aaa')
      console.log(result)
      return 'async函数成功'
  }
  //处理异步执行过程中发生的成功或者失败的逻辑
  test().then((data)=>{   
      console.log(data)   //成功执行此处，data的值为async函数return的值
  },(error)=>{
      console.log(error)  //失败执行此处
  })
```
The result of highlighting is shown as below.
![1589216740(1)](https://user-images.githubusercontent.com/22291419/81590008-1cf04380-93f5-11ea-9c0c-087743b8368a.png)

Then I checked the HTML and then I found there seems has wrong here.
![1589216937(1)](https://user-images.githubusercontent.com/22291419/81590289-92f4aa80-93f5-11ea-9a69-990a6678f9aa.png)

**Comment by yyyc514** [comment]
Closing as duplication of https://github.com/highlightjs/highlight.js/issues/2542.

The fix is in master if you can build from source.

**Comment by yyyc514** [comment]
Resolved in 10.0.3 since technically this was a regression from our last upgrade.
